### PR TITLE
Cowboy bundle

### PR DESCRIPTION
**Pull request by FudgePucker**
Adds a cowboy bundle.
Yes I know there's already a revolver bundle, but does that one come with a cowboy hat?

### CODE REVIEW

**Comment by kol1th** [comment]
nice checks

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -94,7 +94,23 @@
 				new /obj/item/weapon/kitchen/utensil/knife/large/butch/meatcleaver(src)
 				new /obj/item/weapon/storage/pill_bottle/hyperzine(src)
 				new /obj/item/weapon/card/id/syndicate(src)
-
+				
+			if("cowboy")
+				new /obj/item/clothing/accessory/storage/bandolier(src)
+					contents = list()
```
Comment (PJB3005) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:100]: Why is any of this indented, why are you resetting contents like this?

Are you trying to make this code be called on the new bandolier? Sadly that's not how DM works, what you should do is:

``` DM
var/obj/item/clothing/accessory/storage/bandolier/B = new /obj/item/clothing/accessory/storage/bandolier(src)
for (var/i = 1 to 6) // Spawn six speedloaders inside the bandolier.
    new /obj/item/ammo_storage/speedloader/a357(B)

new /obj/item/clothing/suit/storage/det_suit(src)
...
```

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 2]
```diff
@@ -94,7 +94,23 @@
...
+					new /obj/item/ammo_storage/speedloader/a357(src)
+					new /obj/item/ammo_storage/speedloader/a357(src)
+					new /obj/item/ammo_storage/speedloader/a357(src)
+					new /obj/item/ammo_storage/speedloader/a357(src)
+					new /obj/item/ammo_storage/speedloader/a357(src)
+					new /obj/item/ammo_storage/speedloader/a357(src)
```
Comment (ghost) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:106]: 6 speedloaders, plus the rounds in the mateba, makes 42 rounds of ammunition for a gun that can 2-tap a guy

Holy crap

**Comment by kernikov** [comment]
@Skullyton Just call security lmao

**Comment by FudgePucker** [comment]
@PBJ3005
Ah. Thanks. First PR that I made that is more complex than a quick copy paste minor edit.

@Skullyton
Also is a random bundle, also costs 10 TCs.
A single stray taser spells the end, a valid hungry AI can bolt you in. Also no emag like the other bundle.

Rad bow and esword doesn't need ammo.

What would you prefer in a traitor?
The guy walking around with a cowboy hat and revolver getting into a shoot out with the entire station, or that one faggot that sleepy pens his target, c4s him, then does fuck all for two hours?

**Comment by ComicIronic** [comment]
The one that wasn't enabled to murderbone.

**Comment by FudgePucker** [comment]
So you prefer spending two hours as a ghost watching your killer do fuck all instead of watching a western shoot off between a bandito and the grizzled detective?

**Comment by kol1th** [comment]
@FudgePucker Mateba does 60 brute. You kill people in two shots. Two shots is hardly enough time to react with a taser. You gave them more than 50 shots. Why?

**Comment by FudgePucker** [comment]
Fine jeez.

I guess its not high noon.

**Comment by kol1th** [comment]
I love memes.

**Pull request closed without merging**